tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -175,29 +193,53 @@ export interface uploadResourceArgs {
   page: Page
   resources: File[]
   to?: string
-  createVersion?: boolean
+  option?: string
 }
 
 export const uploadResource = async (args: uploadResourceArgs): Promise<void> => {
-  const { page, resources, to, createVersion } = args
+  const { page, resources, to, option } = args
   if (to) {
     await clickResource({ page: page, path: to })
   }
 
   await page.locator(resourceUploadButton).click()
   await page.locator(fileUploadInput).setInputFiles(resources.map((file) => file.path))
 
-  if (createVersion) {
-    await page.locator(actionSecondaryConfirmationButton).click()
-    // @TODO check if upload was successful
-  }
-
-  await page.locator(uploadInfoCloseButton).click()
+  if (option) {
+    switch (option) {
+      case 'Skip': {
+        await page.locator(actionCancelButton).click()
+        break
+      }
+      case 'Replace': {
+        await page.locator(actionSecondaryConfirmationButton).click()
```

Comment:
```suggestion
        await page.locator(actionReplaceButton).click()
```

Reply:
@saw-jan bro sorry but this same selector is used as replace button in duplicate file upload and merge in duplicate folder upload